examples/application/program.teal

```diff
@@ -0,0 +1,39 @@
+#pragma version 6
```

Comment:
@algoidurovic Minor:  I'm unfamiliar with our `examples/application` conventions.  I don't see other `.teal` sources versioned.  

I'm wondering if 1 of the following ought to happen?
* Declare we do _not_ version `teal` sources in `examples/application` by adding a `.gitignore` entry.
* Declare we _want_ to version `teal` sources and either rename to a more specific name (e.g. `opup.teal`) or reorganize applications (e.g. dir per app).
* Delete `program.teal` and deal with the concern another time.

If a policy doesn't exist, I think it's useful to eventually declare a policy.  Though I don't want to hold the PR.

I can be convinced to leave the file as is rather than deleting or renaming it.  Though leaving as is feels crufty to me.

Reply:
Sorry, this file should've been deleted. Taken care of now.

Reply:
@algoidurovic No worries - Thanks for the change in https://github.com/algorand/pyteal/pull/274/commits/a9d22ea6a16fcf4a0acbab00760395668d94b325.  Resolving.